Lib/test/test_urllibnet.py

```diff
@@ -24,8 +24,8 @@ def tearDown(self):
         socket.setdefaulttimeout(None)
 
     def testURLread(self):
-        with support.transient_internet("www.example.com"):
-            f = urllib.request.urlopen("http://www.example.com/")
+        with support.transient_internet("www.pythontest.net"):
```

Comment:
Why not using support.TEST_HTTP_URL here?

Reply:
because `support.transient_internet` uses a hostname and not scheme+hostname.

Reply:
for that, I have introduced `support.TEST_HOSTNAME`